modules/rosa-edit-objects.adoc

```diff
@@ -69,6 +69,8 @@ $ rosa edit cluster --cluster=mycluster --interactive
 [id="rosa-edit-ingress_{context}"]
 == edit ingress
 
+//Does this content need updating with Managed Ingress in 4.13?
+
```

Comment:
Yes, I think that it does @bmcelvee - I think we can just take the fields from the latest (1.2.26) ROSA CLI and put them in here? Although I think they'll likely apply to 4.14 in this context? 
```
[9:28:48] hectorkemp:custom-domains-operator git:(readme-deprecation-legacy-ingress) $ rosa edit ingress -h
Edit a cluster ingress for a cluster.

Usage:
  rosa edit ingress ID [flags]

Aliases:
  ingress, route

Examples:
  # Make additional ingress with ID 'a1b2' private on a cluster named 'mycluster'
  rosa edit ingress --private --cluster=mycluster a1b2

  # Update the router selectors for the additional ingress with ID 'a1b2'
  rosa edit ingress --label-match=foo=bar --cluster=mycluster a1b2

  # Update the default ingress using the sub-domain identifier
  rosa edit ingress --private=false --cluster=mycluster apps

  # Update the load balancer type of the apps2 ingress 
  rosa edit ingress --lb-type=nlb --cluster=mycluster apps2

Flags:
  -c, --cluster string                         Name or ID of the cluster.
      --cluster-routes-hostname string         Components route hostname for oauth, console, download.
      --cluster-routes-tls-secret-ref string   Components route TLS secret reference for oauth, console, download.
      --excluded-namespaces string             Excluded namespaces for ingress. Format should be a comma-separated list 'value1, value2...'. If no values are specified, all namespaces will be exposed.
  -h, --help                                   help for ingress
      --label-match string                     Alias to 'route-selector' flag.
      --lb-type string                         Type of Load Balancer. Options are classic,nlb.
      --namespace-ownership-policy string      Namespace Ownership Policy for ingress. Options are Strict,InterNamespaceAllowed. Default is 'Strict'.
      --private                                Restrict application route to direct, private connectivity.
      --route-selector string                  Route Selector for ingress. Format should be a comma-separated list of 'key=value'. If no label is specified, all routes will be exposed on both routers. For legacy ingress support these are inclusion labels, otherwise they are treated as exclusion label.
      --wildcard-policy string                 Wildcard Policy for ingress. Options are WildcardsDisallowed,WildcardsAllowed. Default is 'WildcardsDisallowed'.

Global Flags:
      --color string     Surround certain characters with escape sequences to display them in color on the terminal. Allowed options are [auto never always] (default "auto")
      --debug            Enable debug mode.
  -i, --interactive      Enable interactive mode.
      --profile string   Use a specific AWS profile from your credential file.
      --region string    Use a specific AWS region, overriding the AWS_REGION environment variable.
```